cinema/serializers.py

```diff
@@ -18,13 +28,26 @@ class Meta:
 class CinemaHallSerializer(serializers.ModelSerializer):
     class Meta:
         model = CinemaHall
-        fields = ("id", "name", "rows", "seats_in_row", "capacity")
+        fields = (
+            "id",
+            "name",
+            "rows",
+            "seats_in_row",
+            "capacity"
```

Comment:
The `capacity` field is included in the `CinemaHallSerializer`. Ensure that the `CinemaHall` model has a `capacity` field or property, as this field is not typically a direct model field.